2.4/root/usr/bin/run-mongod

```diff
@@ -74,26 +74,25 @@ if [ -z "${MONGODB_REPLICA_NAME-}" ]; then
     usage
   fi
   # Run the MongoDB in 'standalone' mode
-  if [ ! -f ${MONGODB_DATADIR}/.mongodb_datadir_initialized ]; then
-    # Create MongoDB users and restart MongoDB with authentication enabled
-    # At this time the MongoDB does not accept the incoming connections.
-    mongod $mongo_common_args & #--bind_ip 127.0.0.1 --quiet >/dev/null &
-    wait_for_mongo_up
+  mongod $mongo_common_args &
+  wait_for_mongo_up
+  js_command="db.system.users.count({'user':'admin'})"
+  if [ "$(mongo admin --quiet --eval "$js_command")" == "1" ]; then
+    echo "=> Admin user is already created. Reseting password ..."
+    mongo_reset_admin
+  else
     mongo_create_admin
-    mongo_create_user
-    touch ${MONGODB_DATADIR}/.mongodb_datadir_initialized
-    # Restart the MongoDB daemon to bind on all interfaces
-    mongod $mongo_common_args --shutdown
-    wait_for_mongo_down
+  fi
+  js_command="db.system.users.count({'user':'${MONGODB_USER}'})"
```

Comment:
What if `MONGODB_USER` is changed?

I think that was not properly supported before, only password changes.
So this seems good, if the value of `MONGODB_USER` is changed, it will create a new user (and the old user is left intact). :+1: